docs/contributing.rst

```diff
@@ -71,15 +71,17 @@ Running the compiler tests
 ==========================
 
 The ``./scripts/tests.sh`` script executes most Solidity tests and
-runs ``aleth`` automatically if it is in the path, but does not download it,
+runs ``aleth`` automatically if it is in the path. The script does not download it,
 so you need to install it first. Please read on for the details.
 
-Solidity includes different types of tests, most of them bundled into the ``soltest``
-application. Some of them require the ``aleth`` client in testing mode, others require ``libz3``.
+Solidity includes different types of tests, most of them bundled into the `C++ Boost Test Framework <https://www.boost.org/doc/libs/1_69_0/libs/test/doc/html/index.html>`_ application ``soltest``.
+Some of them require the ``aleth`` client in testing mode, others require ``libz3``.
 
 To run a basic set of tests that require neither ``aleth`` nor ``libz3``, run
-``./scripts/soltest.sh --no-ipc --no-smt``. This script runs ``./build/test/soltest``
-internally.
+``./scripts/soltest.sh --no-ipc --no-smt``. This script runs ``./${BUILD_DIR}/test/soltest``
+internally where ``${BUILD_DIR}` is set to `build` by default. By setting environment variable ``BUILD_DIR`` accordingly you can choose which build folder to run the tests under.
+
+``./build/test/soltest --help`` has extensive help on all of the options available. See especially `run_test <https://www.boost.org/doc/libs/1_69_0/libs/test/doc/html/boost_test/utf_reference/rt_param_reference/run_test.html>`_ to list specific unit tests, and `report-level <https://www.boost.org/doc/libs/1_69_0/libs/test/doc/html/boost_test/utf_reference/rt_param_reference/report_level.html>`_ give a more detailed report.
```

Comment:
Please break this line, it eases commenting on specific parts of the sentence.

1) it does not list all options, because some of them are just forwarded to soltest
2) "to list specific unit test" - it will actually _run_ specific unit tests, not just list them.

People also like `--progress`.

Reply:
Line is now broken into a list and `show-progress` is mentioned.